### PR TITLE
Add example with a custom theme chart (in the style of The Economist)

### PR DESCRIPTION
**Pull request by paulathevalley**
The Economist uses the [Officina](https://www.myfonts.com/fonts/itc/officina-sans/) typeface, so I picked the closest free font, Fira Sans. 

I got stuck on a few items: 
- changing the tick sizes dynamically; I commented out the code that I think should work
- how to make the red horizontal line stop at the last intersection between the blue line and the `0` tick. 
- how to hide the bottom-most white horizontal tick

(Also I guess these aren’t tutorials. Should this be turned into a tutorial?)

sample:
screen shot 2016-03-20 at 11 20 05 am
 
result:
screen shot 2016-05-05 at 2 06 55 pm

### CODE REVIEW

**Comment by paulathevalley** [comment]
Todo: 
- [ ] Add red rectangle in upper left 
- [ ] Add blue square with 1 in upper right

**Comment by boygirl** [comment]
@paulathevalley you can use functional styles to kill the bottom white gridline like:
```
style={{
  grid: { stroke: (tick) => tick === 0 ? "transparent" : "white" }
}}
```

File: demo/tutorials/custom-theme.jsx [hunk 1]
```diff
@@ -6,83 +6,284 @@ import { VictoryAxis, VictoryLine } from "victory-chart";
...
+      axisOne: {
+        axis: {
+          stroke: BLUE_COLOR,
+          strokeWidth: 0
+        },
+        ticks: {
+          size: 350,
+          /*
+          TODO: Do not show a tick for the start of the domain
+          size: (tick) => tick == "-10" ? 0 : 350,
```
Comment (paulathevalley) [inline demo/tutorials/custom-theme.jsx]: @boygirl I struggled with the functional styles. :( 

Should these actually be nested in `grid` to work?
ie
```jsx
axisOne: { 
  grid: {
    ticks: { 
       size: (tick) => tick == "10" ? 0 : 350;
    }
  }
}
```

Comment (boygirl) [inline demo/tutorials/custom-theme.jsx]: Nope, they shouldn't be nested, but they go off tickValues not the final formatted string, so maybe try `=== 10` instead?

Comment (boygirl) [inline demo/tutorials/custom-theme.jsx]: Or actually it looks like `new Date(1999, 1, 1)` is the value you're trying to match?

Comment (boygirl) [inline demo/tutorials/custom-theme.jsx]: Or I might have a bad false-y check, and that zero is throwing things off. Maybe try a similar thing on styling stroke instead

**Comment by paulathevalley** [comment]
Ok! Ready for re-review.

screen shot 2016-05-31 at 4 26 00 pm

File: demo/tutorials/custom-theme.jsx [hunk 2]
```diff
@@ -6,84 +6,336 @@ import { VictoryAxis, VictoryLine } from "victory-chart";
...
+              style={styles.lineThree}
+            />
+
+            <VictoryLine
+              data={dataSetOne}
+              domain={{
+                x: [new Date(1999, 1, 1), new Date(2016, 1, 1)],
+                y: [-10, 15]
+              }}
+              interpolation="linear"
```
Comment (boygirl) [inline demo/tutorials/custom-theme.jsx]: "linear" is the default interpolation so this prop can go away

File: demo/tutorials/custom-theme.jsx [hunk 3]
```diff
@@ -6,84 +6,336 @@ import { VictoryAxis, VictoryLine } from "victory-chart";
...
+              standalone={false}
+              style={styles.axisTwo}
+            />
+
+            <VictoryAxis
+              orientation="bottom"
+              scale="time"
+              standalone={false}
+              style={styles.axisYears}
+              tickCount={4}
```
Comment (boygirl) [inline demo/tutorials/custom-theme.jsx]: Since you have tickValues, I don't think tickCount is used

File: demo/tutorials/custom-theme.jsx [hunk 4]
```diff
@@ -6,84 +6,336 @@ import { VictoryAxis, VictoryLine } from "victory-chart";
...
+
+            <VictoryAxis dependent
+              domain={[0, 50]}
+              orientation="right"
+              standalone={false}
+              style={styles.axisTwo}
+            />
+
+            <VictoryAxis
+              orientation="bottom"
```
Comment (boygirl) [inline demo/tutorials/custom-theme.jsx]: "bottom" is the default orientation for an independent axis so this prop can be removed

File: demo/tutorials/custom-theme.jsx [hunk 5]
```diff
@@ -6,84 +6,336 @@ import { VictoryAxis, VictoryLine } from "victory-chart";
...
+              lineHeight={1}
+              style={styles.axisOneCustomLabel}
+            >
+              {"-"}
+            </VictoryLabel>
+
+            <VictoryAxis dependent
+              domain={[-10, 15]}
+              offsetX={0}
+              orientation="left"
```
Comment (boygirl) [inline demo/tutorials/custom-theme.jsx]: "left" is the default orientation for a dependent axis, so this prop can be removed

File: demo/tutorials/custom-theme.jsx [hunk 6]
```diff
@@ -6,84 +6,336 @@ import { VictoryAxis, VictoryLine } from "victory-chart";
...
+              style={styles.axisOneCustomLabel}
+            >
+              {"+"}
+            </VictoryLabel>
+
+            <VictoryLabel
+              x={37} y={199}
+              textAnchor="middle"
+              verticalAnchor="end"
+              lineHeight={1}
```
Comment (boygirl) [inline demo/tutorials/custom-theme.jsx]: `lineHeight={1}` is default

File: demo/tutorials/custom-theme.jsx [hunk 7]
```diff
@@ -6,84 +6,336 @@ import { VictoryAxis, VictoryLine } from "victory-chart";
...
+              style={styles.axisOneCustomLabel}
+            >
+              {"+"}
+            </VictoryLabel>
+
+            <VictoryLabel
+              x={37} y={199}
+              textAnchor="middle"
```
Comment (boygirl) [inline demo/tutorials/custom-theme.jsx]: `textAnchor="middle"` is default

File: demo/tutorials/custom-theme.jsx [hunk 8]
```diff
@@ -6,84 +6,336 @@ import { VictoryAxis, VictoryLine } from "victory-chart";
...
-            standalone={false}
-          />
+          >
+            {"An outlook"}
+          </VictoryLabel>
 
           <VictoryLabel
-            x={25} y={40}
+            x={25} y={70}
             textAnchor="start"
```
Comment (boygirl) [inline demo/tutorials/custom-theme.jsx]: `textAnchor="start"` is default

File: demo/tutorials/custom-theme.jsx (hunk 5 above)
Comment (paulathevalley) [inline demo/tutorials/custom-theme.jsx]: This was the only one I could not remove as it moved the axis to the bottom: 

screen shot 2016-05-31 at 4 50 57 pm

**Pull request merged**